src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/TagHelperCompletionProvider.cs

```diff
@@ -70,15 +70,6 @@ public override IReadOnlyList<RazorCompletionItem> GetCompletionItems(RazorCompl
             if (_htmlFactsService.TryGetElementInfo(parent, out var containingTagNameToken, out var attributes) &&
                 containingTagNameToken.Span.IntersectsWith(context.AbsoluteIndex))
             {
-                if ((containingTagNameToken.FullWidth > 1 || containingTagNameToken.Content == "-") &&
```

Comment:
I'm not aware of a scenario where this will break by just removing, as all tests are passing, and manual usage didn't seem odd.

Reply:
Looks like https://github.com/dotnet/razor-tooling/pull/3839 is the PR that introduced that, and there is a link to https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1338261. Can you manually test the scenario from that issue, to ensure we're not re-introducing something here?

Reply:
Verified no change in behavior for this